app/src/main/java/ar/com/wolox/android/example/ui/home/HomePresenter.java

```diff
@@ -1,11 +1,13 @@
 package ar.com.wolox.android.example.ui.home;
 
 import javax.inject.Inject;
+
```

Comment:
delete this blank line

Reply:
This line is generated by Android Studio when I reformat code